crates/rolldown/tests/rolldown/function/external/double-namespace-reexport/artifacts.snap

```diff
@@ -0,0 +1,26 @@
+---
+source: crates/rolldown_testing/src/integration_test.rs
+snapshot_kind: text
+---
+# Assets
+
+## main.js
+
+```js
+import "external";
+
+
+//#region second.js
+var second_exports = {};
+import * as import_external from "external";
```

Comment:
Here the import should be at top of file to instead of `import "external";` I will fixed it at next.

Reply:
Not needed. It's expected behavior and the same as esbuild did it https://hyrious.me/esbuild-repl/?version=0.23.0&b=e%00entry.js%00import+%27.%2Ffoo%27%0A%0Aexport+%7B+foo+%7D+from+%27.%2Ffirst.js%27%3B&b=%00first.js%00export+*+from+%27external%27%3B&b=%00foo.js%00console.log%28%27foo%27%29&o=--bundle+--external%3Aexternal+--format%3Desm

Reply:
But the top of `import 'external'` is unnecessary. Other named import from external is placed at top, i think it is better if  make them consistent.

Reply:
> But the top of import 'external' is unnecessary.

You could try to solve this one. But `import * as import_external from "external";` is in the right place, don't require to fix it.